eduserver/cache/cache.go

```diff
@@ -109,6 +113,17 @@ func (t *EDUCache) AddTypingUser(
 	return t.GetLatestSyncPosition()
 }
 
+// AddSendToDeviceMessage increases the sync position for
+// send-to-device updates.
+// Returns the latest sync position for typing after update.
```

Comment:
Unsure if this is a typo or not given it does appear to increment the same counter. Needs additional commentary.